modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-syslog_{context}"]
+= Sending logs using the Fluentd syslog plug-in (RFC 3164)
+
+You can use the Fluentd *syslog* plug-in to send a copy of your logs to an external syslog server, 
+not the default Elasticsearch. Note the following about the *syslog* plug-in:
+
+* uses syslog protocol (RFC 3164), not RFC 5424;
```

Comment:
For lists, IBM (p. 80) states:

> Write lists so that all or none of the items start with complete sentences. If list items comprise only complete sentences, include a period after each sentence. If list items comprise only sen- tence fragments, do not include any end punctuation.
> If a list contains list items that start with sentence fragments, and one or more of the fragments are followed by complete sentences, include periods after all sentence fragments and sentences.